homeworks/rostyslav.havuka_RostyslavHavuka/11-dependency-injection/vehicle/src/app/app.component.html

```diff
@@ -0,0 +1,11 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+<mat-spinner *ngIf="areUsersLoading" style="margin: auto; margin-top: 10%;"></mat-spinner>
+
+<ul *ngIf="!areUsersLoading">
+  <li *ngFor="let car of cars" class="mat-subheading-2">
+   <a class="mat-raised-button" href="https://www.google.com/search?q={{ car.vehicle }}" http>Added from <a href="https://www.google.com/search?q={{ car.vehicle }}">{{ car | protocolCar }}</a></a><br>
+   Vehicle: {{ car | infoCar }} <br>
+   Price: {{ car.cost.amount | currency }}
+    {{ car.cost.currencyCode}}
```

Comment:
Currency pipe used incorrect. Read the doc for further info:
https://angular.io/api/common/CurrencyPipe

In short, you need to pass `currencyCode` as the second argument 😄